mobile-widgets/statsmanager.cpp

```diff
@@ -1,31 +1,33 @@
 // SPDX-License-Identifier: GPL-2.0
 #include "statsmanager.h"
+#include "stats/chartlistmodel.h"
 
-StatsManager::StatsManager() : view(nullptr)
+StatsManager::StatsManager() : view(nullptr), charts(nullptr)
 {
-	// Test: show some random data. Let's see what happens.
-	state.var1Changed(2);
-	state.var2Changed(3);
-	state.binner2Changed(2);
+	// Default chart is dives per year
+	state.var1Changed(0);
```

Comment:
If you want the default, you can remove these three lines.

Reply:
ahh, didn't realize that. will do